src/axom/sidre/core/Group.cpp

```diff
@@ -2111,9 +2131,33 @@ void Group::importFrom(conduit::Node& node,
     conduit::NodeIterator groups_itr = node["groups"].children();
     while (groups_itr.has_next())
     {
+      Group* group;
       Node& n_group = groups_itr.next();
+      bool create_list = false;
+      if (n_group.has_child("views"))
+      {
+        if (n_group["views"].dtype().is_list())
+        {
+          create_list = true;
+        }
+      }
+      if (!create_list && n_group.has_child("groups"))
+      {
+        if (n_group["groups"].dtype().is_list())
+        {
+          create_list = true;
+        }
+      }
       std::string group_name = groups_itr.name();
-      Group* group = createGroup(group_name);
+      if (!m_is_list)
```

Comment:
I prefer to avoid negative logic if possible. So my preference is to reverse the clauses so the test is `if (m_is_list)`.  More of a style issue.